ckanext/canada/schemas/presets.yaml

```diff
@@ -1845,8 +1845,8 @@ presets:
   values:
     field_name: time_period_coverage_start
     help_text:
-      en: The start date of the period covered by the dataset
-      fr: Date de début de la période couverte par le produit (AAAA-MM-JJ)
+      en: The start date of the period covered by the dataset (YYYY-MM-DD)
+      fr: Date de début de la période couverte par le jeu de données (AAAA-MM-JJ)
```

Comment:
I know the client wants the (YYY-MM-DD) text but that conflicts with modern browsers that render their own date fields so I tend to remove it